tests/test_player.py

```diff
@@ -0,0 +1,214 @@
+"""
+Unit Tests for Player Model
+
+These tests may fail at any time due to the whims of the Blaseball API. See also: Blaseball Falsehoods
```

Comment:
let's make this not a thing and just use vcr https://vcrpy.readthedocs.io/en/latest/